include/zephyr/drivers/spi.h

```diff
@@ -705,6 +816,7 @@ __subsystem struct spi_driver_api {
 
 /**
  * @brief Check if SPI CS is controlled using a GPIO.
+ * @addtogroup spi_gpio_cs
```

Comment:
Related to previous comment, I don't think one can add to a `@name`'d group 
Doxygen coverage also seems confused by this https://builds.zephyrproject.io/zephyr/pr/86900/api-coverage/zephyr/drivers/spi.h.gcov.html

Reply:
ok ill try to fix the doxygens